example/jaeger/main.go

```diff
@@ -45,18 +45,15 @@ func initTracer() func() {
 		log.Fatal(err)
 	}
 
-	// For demoing purposes, always sample. In a production application, you should
-	// configure this to a trace.ProbabilitySampler set at the desired
-	// probability.
-	tp, err := sdktrace.NewProvider(
-		sdktrace.WithConfig(sdktrace.Config{DefaultSampler: sdktrace.AlwaysSample()}),
-		sdktrace.WithSyncer(exporter))
-	if err != nil {
-		log.Fatal(err)
-	}
-	global.SetTraceProvider(tp)
+	// For demoing purposes, use trace.AlwaysSample to sample every trace.
+	// In a production application, you should configure this to a trace.ProbabilitySampler
+	// set at the desired probability.
+	tp.ApplyConfig(sdktrace.Config{
```

Comment:
I would prefer to see the `sdktrace.Config` argument passed directly to `InstallNewPipeline`. Any reason not to?

Reply:
Well, I guess it depends on what is expected of `NewExportPipeline` / `InstallNewPipeline` - as this is supposed to be a simplified setup and there are default values which will be used if no config is passed to the trace provider, it is capable of returning pipeline with default configuration without needing this extra parameter.

On the other hand, I guess _default_ does not necessarily correspond to _recommended_ (and universally recommended config might not exist in this case), which is probably what the simplified pipeline setup should actually do for the user, so providing config would be reasonable as well. I was considering both implementations, so I'm not opposed.

Reply:
we're already passing in other Jaeger properties to InstallNewPipeline, just not sdktrace.Config... so I'm on the fence here.

Reply:
Mixing functional options with a option struct seems a little weird. 

I would love to see an unified functional options for exporters that could configure the underline SDK as well but I don't know if it is feasible without some adaptations. 

Something along the lines should be possible and I think it is better than passing the Options struct directly:
```go
// Create and install Jaeger export pipeline
tp, flush, err := jaeger.InstallNewPipeline(
	jaeger.WithCollectorEndpoint("http://localhost:14268/api/traces"),
	jaeger.WithProcess(jaeger.Process{
		ServiceName: "trace-demo",
		Tags: []core.KeyValue{
			key.String("exporter", "jaeger"),
			key.Float64("float", 312.23),
		},
	}),
	jaeger.WithSDK(
		sdktrace.AlwaysSample(),
		sdktrace.WithDefaultLabelEncoder()
	),
)
```